src/cpu/aarch64/brgemm/brgemm.cpp

```diff
@@ -461,7 +461,7 @@ status_t brgemm_init_tiles(const brgemm_t &brg, char palette[64]) {
 
 namespace {
 template <typename T>
-static inline int sign(T v) {
+inline int sign(T v) {
```

Comment:
What's the rationale on these ones where we remove static?

Reply:
It's tucked behind `unnamed namespace`, which makes this function `static` by meaning, no need for extra redundant qualifier.